x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/InsistParameters.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.plan;
+
+import org.elasticsearch.common.io.stream.NamedWriteable;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+// To make things simpler, we're only dealing with a single INSIST parameter for now.
+public record InsistParameters(String identifier) implements NamedWriteable {
```

Comment:
I don't see the need for a separate class just to hold a list of field names (assuming that this will be in the future a list).